homeworks/nikita.karchinskiy_Nikkarchinskiy/4-moduls/exports.js

```diff
@@ -0,0 +1,52 @@
+const monthIs = document.querySelector('[data-mounth-long-answer]');
+const monthMondays = document.querySelector('[data-mondays-answer]');
+const shortWeeks = document.querySelector('[data-shortest-week-answer]');
+
+export function daysInMonthFnc(isYear, isMonth) {
+    const daysInMonth = new Date(isYear, isMonth, 0).getDate();
+    if (daysInMonth < 31) {
+        monthIs.insertAdjacentHTML('beforeend', 'long');
+        console.log(1);
```

Comment:
It seems like you don't need these console logs. So it would help if you removed them